tests/functional/scripts/pyi_lib_boto.py

```diff
@@ -0,0 +1,45 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2015, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+from inspect import getmembers, isfunction
+from functools import partial
+import boto
+import boto.exception
+
+credentials = dict(
+    aws_access_key_id='ASIAIH3F2FU3T63KIXKA',
+    aws_secret_access_key='lnN4qk1a0SuQAFVsGA+Y+ujo2/5rLq2j+a1O4Vuy'
```

Comment:
Those are fake values. There is some validator that was causing an error like "no auth method available" if I didn't use values of a valid format.